catalog/data_catalogs.go

```diff
@@ -0,0 +1,25 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceCatalogs() *schema.Resource {
+	var data struct {
```

Comment:
I would better go with map of name to ID or vice versa

Reply:
Id of catalog is the name